src/Compilers/CSharp/Portable/Utilities/TypeSymbolExtensions.cs

```diff
@@ -43,6 +43,17 @@ public static int CustomModifierCount(this TypeSymbol type)
                         TypeWithAnnotations pointedAtType = pointer.PointedAtTypeWithAnnotations;
                         return pointedAtType.CustomModifiers.Length + pointedAtType.Type.CustomModifierCount();
                     }
+                case SymbolKind.FunctionPointer:
+                    {
+                        var funcPtr = (FunctionPointerTypeSymbol)type;
+                        var count = funcPtr.Signature.ReturnTypeWithAnnotations.CustomModifiers.Length + funcPtr.Signature.ReturnType.CustomModifierCount();
```

Comment:
>funcPtr.Signature.ReturnTypeWithAnnotations.CustomModifiers.Length + funcPtr.Signature.ReturnType.CustomModifierCount() [](start = 36, length = 119)

I would have expected `funcPtr.Signature.ReturnTypeWithAnnotations.CustomModifiers` to be the same as `funcPtr.Signature.ReturnType.CustomModifierCount()`, so I feel we may be double-counting. Is that not the case?